scripts/test_run_release_server.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+# this script runs the server in release mode and tries to set up a dev environment, which catches failures between the
+# server and CLI, and ensures clap/etc rules actually work
+#
+# you really really really really don't want to run this when an environment you like exists, it'll mess it up
+
+set -e
+
+WAIT_TIMER=5
+
+echo "Building release binaries..."
+cargo build --release --bin kanidm --bin kanidmd
+
+if [ -d '.git' ]; then
+    echo "You're in the root dir, let's move you!"
+    CURRENT_DIR="$(pwd)"
+    cd server/daemon/ || exit 1
+fi
+
+if [ ! -f "run_insecure_dev_server.sh" ]; then
+    echo "I'm not sure where you are, please run this from the root of the repository or the server/daemon directory"
+    exit 1
+fi
+
+mkdir -p /tmp/kanidm/
+
+echo "Generating certificates..."
+cargo run --bin kanidmd --release cert-generate --config ../../examples/insecure_server.toml
+echo "Running the server..."
+cargo run --bin kanidmd --release server --config ../../examples/insecure_server.toml &
```

Comment:
What's the error condition here? Shouldn't we try an admin pw recovery and use the success/failure of that to determine is this is a pass/fail?

Reply:
It starts the server so the setup dev environment script can work so it tests "live" database and web things... which we've missed in testing before 😨

Reply:
Yeah, but my point here is that if the server fails to start do we detect or error here in this script?

Reply:
Ah... I'll double check that.

Reply:
Sorted... I think!